docs/content/querying/segmentmetadataquery.md

```diff
@@ -96,12 +97,16 @@ This is a list of properties that determines the amount of information returned
 
 By default, all analysis types will be used. If a property is not needed, omitting it from this list will result in a more efficient query.
 
-There are 3 types of column analyses:
+There are 4 types of column analyses:
 
 #### cardinality
 
 * Estimated floor of cardinality for each column. Only relevant for dimension columns.
 
+#### min_max
```

Comment:
@xvrl The json value for `AnalysisType` is `name().toLowerCase()`. so I've changed it to `minmax` for consistency (not `minMax`). Would it be ok?